resources/qml/qmldir

```diff
@@ -35,6 +35,7 @@ RadioButton         1.0 RadioButton.qml
 Scrollable          1.0 Scrollable.qml
 TabButton           1.0 TabButton.qml
 TextField           1.0 TextField.qml
+ScrollView          1.0 ScrollView.qml
```

Comment:
Super nitpicky, shouldn't it be 1.1? You actually changed it in AddLocalPrinterScrollView.qml to use cura 1.1 instead of 1.0

Reply:
To be honest, I don't know what the version of the Cura namespace should be. The change to Cura 1.1 is a vestigial change from @diegopradogesto's work. 

I'm not even sure if registering ScrollView in qmldir is necessary; neither Scrollable.qml nor TabButton.qml exist in the current codebase, and their replacements are not registered but still work.

Personally, given this nitpick, I would change AddLocalPrinterScrollView.qml back to using Cura 1.0. Let me know what you prefer.

Reply:
Oh, well. I don't think it's very actively used by us. The idea is to have versioning in your qml. 

I thought it was an intentional change, hence the nitpick. If it doesn't serve a real purpose, it doesn't really matter which one is used.